README.md

```diff
@@ -64,53 +70,36 @@ run()
 
 See the [toolkit documentation](https://github.com/actions/toolkit/blob/master/README.md#packages) for the various packages.
 
-## Package for distribution
-
-GitHub Actions will run the entry point from the action.yml. Packaging assembles the code into one file that can be checked in to Git, enabling fast and reliable execution and preventing the need to check in node_modules.
-
-Actions are run from GitHub repos.  Packaging the action will create a packaged action in the dist folder.
-
-Run prepare
-
-```bash
-npm run prepare
-```
+## Publish to a distribution branch
 
-Since the packaged index.js is run from the dist folder.
+Actions are run from GitHub repos so we will checkin the packed dist folder. 
 
+Then run [ncc](https://github.com/zeit/ncc) and push the results:
 ```bash
-git add dist
-```
-
-## Create a release branch
-
-Users shouldn't consume the action from master since that would be latest code and actions can break compatibility between major versions.
-
-Checkin to the v1 release branch
-
-```bash
-git checkout -b v1
-git commit -a -m "v1 release"
-```
-
-```bash
-git push origin v1
+$ npm run package
+$ git add dist
+$ git commit -a -m "prod dependencies"
+$ git push origin releases/v1
```

Comment:
don't really understand this; can you check over grammar

Also maybe git commit -a -m "chore: Distribute action"